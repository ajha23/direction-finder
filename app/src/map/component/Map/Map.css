/* Always set the map height explicitly to define the size of the div
 * element that contains the map. */
 #map {
    height: 100%;
  }
  .map-container {
    width: calc(100% - 32px);
    padding: 16px;
    height: 500px;
}

.map-container > div {
    width: 100%;
    height: 100%;
}