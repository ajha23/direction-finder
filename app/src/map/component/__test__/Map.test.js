import React from 'react';
import renderer from 'react-test-renderer';

import Map from '../Map/Map';

describe('Direction Map Test', () => {
    it('render correctly', () => {
        const tree = renderer.create(<Map />).toJSON();

        expect(tree).toMatchSnapshot();
    });
});