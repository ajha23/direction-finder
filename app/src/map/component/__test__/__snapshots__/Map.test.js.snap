// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Direction Map Test render correctly 1`] = `
<div
  className="map-container"
>
  <div />
</div>
`;
